script/testing/junit/.gitignore

```diff
@@ -1,4 +1,2 @@
-
-!Makefile
-!lib/*.jar
```

Comment:
Is there a reason for these changes?

Reply:
- the changes on these 2 lines are optional
- all i did is to remove the `!Makefile` gitignore negation since there is no `Makefile` needed in the junit testing
- the removal of the `!out/` is necessary nevertheless

Reply:
ok sweet